uiowa/uiowa
# [IWP] Media gallery migration

Resolves #7920





# How to test


```
ddev blt ds --site=iwp.uiowa.edu && ddev drush @iwp.local en iwp_migrate -y && ddev drush @iwp.local uli /admin/config/sitenow/sitenow-migrate
```

- Configure SiteNow Migrate following instructions for DevDesktop. 
  - Database: `iwp`.
  - User: `root`
  - Host: `host.docker.internal`
  - Port: `33067`
  - Base URL: `https://iwp.uiowa.edu`
- Check the migration status: `ddev drush @iwp.local ms`
- Run all migrations `ddev drush @iwp.local mim --all  `  
- or
- Import the media gallery: `ddev drush @iwp.local mim iwp_gallery`
- Import the redirects: `ddev drush @iwp.local mim iwp_gallery_redirects`
- Rollback `ddev drush @iwp.local mr iwp_gallery`

Go to https://iwp.uiowa.ddev.site/media-gallery to view all. 
Check against live site to make sure things migrated over correctly
